crates/ruff/src/rules/pandas_vet/rules/inplace_argument.rs

```diff
@@ -8,6 +8,26 @@ use crate::rules::pandas_vet::fixes::fix_inplace_argument;
 use crate::violation::AlwaysAutofixableViolation;
 
 define_violation!(
+    /// ## What it does
+    /// Checks for `inplace=True` in code using the `pandas` library.
+    /// 
+    /// ## Why is this bad?
+    /// - Many people expect `inplace=True` to be a performance benefit that prevents dataframe copies, but that's often not true.
```

Comment:
I think the claim "Many people expect" should be removed ... we don't know what people expect, I think just saying that it often doesn't yield performance benefits should suffice. It would be interesting to note why it often doesn't improve performance.

Reply:
Removed the claim.